subcommands/waves/init.go

```diff
@@ -145,6 +150,31 @@ func doInitWave(cmd *cobra.Command, args []string) {
 	} else {
 		subcommands.DieNotNil(api.FactoryCreateWave(factory, &wave), "Failed to create a wave")
 	}
+
+	if !hasStaticDelta(new_targets) {
+
+		fmt.Print(`
+WARNING: You created a wave for a target version without static deltas.
+
+We recommend that you generate static deltas for your production targets to optimize
+the size of the update. Please, consider generating a static delta for targets using:
+$ fioctl targets static-deltas.
+
+You can then cancel this wave and create a new one for a target with a static delta.
+`)
+
+	}
+}
+
+func hasStaticDelta(new_targets tuf.Files) bool {
+	for _, file := range new_targets {
+		custom, err := api.TargetCustom(file)
+		subcommands.DieNotNil(err)
```

Comment:
I am not sure we need to "die" in the case of error since printing the warning message about static delta is not mandatory. Maybe, just warning would suffice.

Reply:
I am on the fence here.

From one perspective, we can say the user that we were not able to check if the wave has static deltas.

But, this clearly means that the JSON structure is invalid.
To make matters worse, we've already verified it to be valid earlier.
Thus, it is definitely an error.

So, I _guess_ it is better to let the user know that we have a bug in our code by shouting out loud.

Reply:
ok, and it makes the implementation simpler.

Reply:
Cool closing this one as done.
thx